<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(104, 135, 199, 255), stop:1 rgba(16, 38, 84, 255));
}</string>
  </property>
  <widget class="QLabel" name="current_temp">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>141</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>40</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>26°C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="current_picture">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>95</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel 
{
	background: none;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sunny.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="native_city_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>481</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>40</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Сочи</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="current_description">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>184</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>облачно с прояснениями</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="current_humidity">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>161</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Влажность: 55%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="current_feels_like">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>105</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Ощущается как 36°C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="current_speed">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>155</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Скорость ветра: 6.26 м/с</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="current_pressure">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Давление: 750 мм.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>460</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	border-radius: 20;
	background-color: rgb(229, 229, 248, 160);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>460</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	border-radius: 20;
	background-color: rgb(229, 229, 248, 110);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>460</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background-color: rgb(229, 229, 248, 60);
	border-radius: 20
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>460</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background-color: rgb(229, 229, 248, 60);
	border-radius: 20
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="current_time_weekday">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>16:30, Ср.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>1</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background-color: rgb(229, 229, 248, 90);
	border-radius: 20
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>240</y>
     <width>1</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background-color: rgb(229, 229, 248, 90);
	border-radius: 20
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nearest_1_picture">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>250</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel 
{
	background: none;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sunny.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nearest_1_temp">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>24°C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="nearest_1_time">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>290</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>18:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="nearest_2_temp">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>21°C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="nearest_2_picture">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>250</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel 
{
	background: none;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>cloudy.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nearest_2_time">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>290</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>21:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="nearest_3_temp">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>19°C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="nearest_3_picture">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>250</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel 
{
	background: none;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>snowing.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nearest_3_time">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>290</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="day_1_picture">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel 
{
	background: none;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>raining.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="day_1_weekday_button">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>377</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background: rgb(207, 213, 255, 100);
	color: rgb(255, 255, 255);
	border-radius: 15;
}</string>
   </property>
   <property name="text">
    <string>Чт</string>
   </property>
  </widget>
  <widget class="QLabel" name="day_1_min_max_temp">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>370</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>-18°C -&gt; -26°C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="day_1_speed">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>370</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>4 м/с</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="day_2_picture">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>480</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel 
{
	background: none;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>raining.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="day_2_weekday_button">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>477</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background: rgb(207, 213, 255, 100);
	color: rgb(255, 255, 255);
	border-radius: 15;
}</string>
   </property>
   <property name="text">
    <string>Чт</string>
   </property>
  </widget>
  <widget class="QLabel" name="day_2_speed">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>4 м/с</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="day_3_picture">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>580</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel 
{
	background: none;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>raining.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="day_3_weekday_button">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>577</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background: rgb(207, 213, 255, 100);
	color: rgb(255, 255, 255);
	border-radius: 15;
}</string>
   </property>
   <property name="text">
    <string>Чт</string>
   </property>
  </widget>
  <widget class="QLabel" name="day_3_speed">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>570</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>4 м/с</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="day_2_min_max_temp">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>470</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>-18°C -&gt; -26°C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="day_3_min_max_temp">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>570</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara Light</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: none;color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>-18°C -&gt; -26°C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
